src/go/devcontainer-feature.json

```diff
@@ -15,6 +15,14 @@
             ],
             "default": "latest",
             "description": "Select or enter a Go version to install"
+        },
+        "golangciLintVersion": {
+            "type": "string",
+            "proposals": [
```

Comment:
Could we add any other `proposals`? , if not, then feel free to drop this property.

Reply:
The linter gets upgraded too frequently to list proposed versions IMHO. I'll just remove the property